src/raftstore/store/store.rs

```diff
@@ -1453,6 +1454,22 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.register_compact_lock_cf_tick(event_loop);
     }
 
+    fn on_dump_rocksdb_metrics(&mut self, event_loop: &mut EventLoop<Self>) {
```

Comment:
same reason, it is not suitable to get rocksdb statistics in raftstore mod. 
You can use a function to get all rocksdb statistics, then use this function here.